.github/workflows/ci.yml

```diff
@@ -46,6 +46,8 @@ jobs:
         with:
           path: node_modules
           key: ${{ runner.os }}-nextjs-${{ hashFiles('package-lock.json') }}
+      - name: Prisma generate
+        run: npx prisma generate
```

Comment:
Was the type checking failing without this?

Reply:
Yes, the type check was failing before